src/components/algolia-search/lib/get-items-exist.ts

```diff
@@ -0,0 +1,18 @@
+import { AutocompleteCollection, BaseItem } from '@algolia/autocomplete-core'
+
+/**
+ * If any collection has hit 'items', return true. This is used to help
+ * determine whether the 'no results' state should show
+ */
+export function getItemsExist(collections: AutocompleteCollection<BaseItem>[]) {
```

Comment:
Maybe `getHasResults` is a better name?... 💭 I don't love this name.

Reply:
💭 What do you think of just `hasResults`?